src/app/admin/dashboard-kpi/dashboard-kpi.component.ts

```diff
@@ -0,0 +1,169 @@
+import { Component, OnInit, ElementRef } from '@angular/core';
+import { UsersService } from "../users/services/users.service"; 
+import { ChartOptions, ChartType, ChartData, ChartDataSets } from 'chart.js';
+import { Label } from 'ng2-charts';
+import activeInactiveUser from "../../shared/model/activeInactiveUser";
+ 
+@Component({
+  selector: 'app-dashboard-kpi',
+  templateUrl: './dashboard-kpi.component.html',
+  styleUrls: ['./dashboard-kpi.component.scss']
+})
+export class DashboardKpiComponent implements OnInit {
+
+  users: activeInactiveUser[] = [];
+  newUsers: number[] = [];
+  numberNewUsers: number[] = [];
+  userMonth: string[] = [];
+  numberActiveUsers: number[] = [];
+  numberInactiveUsers: number[] = [];
+
+  barChartOptions: ChartOptions = {
+    responsive: true,
+    scales: {
+      yAxes: [{
+        scaleLabel: {
+          display: true,
+          labelString: 'Usuários'
+        },
+        ticks: {
+          precision: 0,
+        }
+      }]
+    },
+    legend: {
+      position: 'right',
+    }
+  };
+  barChartLabels: Label[] = this.months();
+  barChartType: ChartType = 'bar';
+  barChartLegend = true;
+  barChartPlugins = [];
+  barChartData: ChartDataSets[] = [{ 
+      label: 'Ativos', 
+      data: this.activeUsers(),
+      backgroundColor: 'rgb(48, 63, 159)',
+    },
+    {
+      label: 'Inativos',
+      data: this.inactiveUsers(),
+      backgroundColor: 'rgb(255, 0, 0)',
+    }
+  ];
+
+  lineChartOptions: ChartOptions = {
+    responsive: true,
+    scales: {
+      yAxes: [{
+        scaleLabel: {
+          display: true,
+          labelString: 'Usuários',
+        },
+        ticks: {
+          precision: 0,
+        }
+      }]
+    },
+    legend: {
+      position: 'right',
+    }
+  };
+  lineChartLabels: Label[] = this.months();
+  lineChartType: ChartType = 'line';
+  lineChartData: ChartDataSets[] = [{ 
+      label: 'Novos', 
+      data: this.usersByMonthAndYear(),
+      fill: false,
+      borderColor: 'rgb(48, 63, 159)',
+      pointBackgroundColor: 'rgb(48, 63, 159)',
+    }
+  ];
+
+  constructor(private userService: UsersService, private elementRef: ElementRef) { }
+
+  ngOnInit(): void { 
+    
+  }
+
+  months() {
```

Comment:
Função pode ser escrita de forma mais suscinta, colocando os meses dentro de um vetor, e acessando o mês desejado através do índice
Exemplo:
`
const monthNames = ["January", "February", "March", "April", "May", "June",
  "July", "August", "September", "October", "November", "December"
];

const d = new Date();
document.write("The current month is " + monthNames[d.getMonth()]);
`